roles/adoption_osp_deploy/tasks/deploy_ceph.yml

```diff
@@ -0,0 +1,71 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Create block devices on compute nodes using ceph playbook
```

Comment:
in our case, we can even go with actual drives (via the extra_disks parameter in the VMs description). Would be faster.

Reply:
I started by porting what we currently have, but yes, we should align it better to what we currently have in the VA

Reply:
the last version uses extra_disks instead of the block devices, it seems to work well